src/components/utils/crossDeviceSync.js

```diff
@@ -0,0 +1,9 @@
+import io from 'socket.io-client'
+
+export const socketIoConfig = () =>
```

Comment:
Minor, but this isn't really the `socketIoConfig`. It's the actual socket generator.

Maybe `export const createSocket = () =>` or something?